packages/create-turbo/src/commands/create/index.ts

```diff
@@ -85,12 +93,20 @@ export async function create(
 
   const { example, examplePath } = opts;
   const exampleName = example && example !== "default" ? example : "basic";
-  const { hasPackageJson, availableScripts, repoInfo } = await createProject({
-    appPath: root,
-    example: exampleName,
-    isDefaultExample: isDefaultExample(exampleName),
-    examplePath,
-  });
+
+  let projectData = {} as Awaited<ReturnType<typeof createProject>>;
+  try {
+    projectData = await createProject({
```

Comment:
This eventually hits `got` with no timeout: https://github.com/vercel/turbo/blob/main/packages/turbo-utils/src/examples.ts#L48C34-L50

The failure mode for the last outage didn't return a 500 unicorn, it just hung indefinitely.

This is definitely an improvement, but it doesn't address all failure modes.